### PR TITLE
Refactor SnykToolWindowPanel unit test to not use jetbrains test base class

### PR DESCRIPTION
**Pull request by bastiandoetsch**
also contains the commit to only enable experiments if tracking is enabled

### CODE REVIEW

File: src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt
Comment (ArtsiomCh) [inline src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt:79]: wow! didn't know that trick, how did you found it? can't find it in official docs....

File: src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt
Comment (ArtsiomCh) [inline src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt:81]: wow! didn't know that trick. How did you found it? Can't find it in official docs....

Comment (bastiandoetsch) [reply src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt:81]: I just wrote it as I would be writing it in shell scripting, and surprisingly it worked...

**Comment by bastiandoetsch** [comment]
bump @ArtsiomCh @pavel-snyk . Please review (again).

**Comment by ArtsiomCh** [comment]
As I said before, I'm against mocking Jetbrains classes and if that needed, imho, JB test framework should be used. So I'm not going to use that testing approach even if merged. Would be not happy to fix/change such tests when introducing new functionality, but I'm ok with that if @pavel-snyk found it useful.

**Comment by bastiandoetsch** [comment]
@pavel-snyk please comment - btw., without this change, we would keep a test inheriting from LightPlatformTestCase in Unit tests.

**Review by ArtsiomCh — CHANGES_REQUESTED** [review]
Besides I'm still against that approach, I feel @bastiandoetsch you need that for your TTD developing way. 
I'm willing to approve it after merge conflict resolving (and other code clean up if needed to match current `master` state)
PS Please use that mocks with caution and always think about fellow developers who will need to adjust such tests later.

**Comment by bastiandoetsch** [comment]
I won't merge it now - it is failing now after rebasing with master. And the exception is very weird, so as it's not really and very important, I'll let it rest.

**Pull request closed without merging**
